### PR TITLE
Improve api response

### PR DESCRIPTION
**Pull request by cp-sumi-k**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [ ] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [ ] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [ ] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [ ] Remove unused or commented code if not required
- [ ] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [ ] Inspector (all iPhones and android devices like samsung, motorola)
- [ ] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [ ] Available android devices (chrome, firefox)
- [ ] iPad(safari)
- [ ] Tablet (chrome, firefox)

### CODE REVIEW

File: nuxt-frontend/components/home-new/BlogSection.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
           Well, at least the community says Hell Yeah. Our blogs hosted on
           medium have
           <span
-            class="secondary-color sub-h1-semibold lg:mobile-header-2-semibold"
+            class="v2-canopas-gradient-text sub-h1-semibold lg:mobile-header-2-semibold"
```
Comment (cp-jagruti-a) [inline nuxt-frontend/components/home-new/BlogSection.vue:19]: Why changed color here?

File: nuxt-frontend/components/home-new/BlogSection.vue [hunk 2]
```diff
@@ -87,56 +84,70 @@
...
+                            : 'bg-[length:0%]'
+                        "
+                        class="sub-h1-semibold lg:mobile-header-2-semibold xl:desk-header-3 text-black-87 cursor-pointer"
+                      >
+                        {{ blog.title }}</span
+                      >
+                    </nuxt-link>
                   </div>
                   <div
                     class="flex flex-row justify-between mt-[0.9375rem] md:mt-4 lg:mt-8 text-black-60"
```
Comment (cp-dharti-r) [inline nuxt-frontend/components/home-new/BlogSection.vue:109]: remove `flex-row`

File: nuxt-frontend/components/home-new/BlogSection.vue (hunk 1 above)
Comment (cp-sumi-k) [reply nuxt-frontend/components/home-new/BlogSection.vue:19]: Its showing error of color contrast ratio on SEO. Its fine to use this

File: nuxt-frontend/pages/tag/[slug].vue [hunk 3]
```diff
@@ -1,7 +1,26 @@
 <template>
   <div>
     <Header />
-    <TagList :slug="slug" :posts="posts" :mixpanel="$mixpanel" />
+    <section
+      v-if="slug !== ''"
+      class="container min-h-[50vh] my-10 md:my-16 sm:mx-auto 3xl:px-24"
+    >
+      <div class="md:mx-8 xl:mx-20">
+        <div class="flex flex-row space-x-4 items-center">
```
Comment (cp-dharti-r) [inline nuxt-frontend/pages/tag/[slug].vue:9]: remove `flex-row`

File: nuxt-frontend/pages/tag/[slug].vue [hunk 4]
```diff
@@ -1,7 +1,26 @@
...
+          <div class="w-6 h-6 md:w-7 md:h-7 mt-1">
+            <Icon name="fa6-solid:tags" class="w-full h-full" />
+          </div>
+          <h1
+            class="my-6 md:my-10 text-[1.5rem] md:text-[1.875rem] xl:text-[2.25rem] leading-7 tracking-none capitalize font-inter-semibold"
```
Comment (cp-dharti-r) [inline nuxt-frontend/pages/tag/[slug].vue:14]: text-[1.5rem] = text-2xl
text-[1.875rem] = text-3xl
text-[2.25rem] = text-4xl

**Review by cp-dharti-r — DISMISSED** [review]

**Pull request merged**